sdk/storage/tests.yml

```
@@ -26,6 +26,12 @@ stages:
         MatrixReplace:
           # Use dedicated storage pool in canadacentral with higher memory capacity
           - Pool=(.*)-general/$1-storage
+        ${{ if contains(variables['Build.DefinitionName'], 'tests-weekly') }}:
+          AdditionalMatrixConfigs:
```

```suggestion
          MatrixConfigs:
```

I forgot we want to override the base matrix, not add on to it.